spec/swagger/definitions/hand_coded_paths_v2.yaml

```diff
@@ -110,3 +110,131 @@
           application/vnd.api+json:
             schema:
               $ref: post_people_responses.yaml#/422
+/people/{person_id}/profiles:
```

Comment:
I guess /api/v2 is shared by a prefix in the overall document :)

Reply:
Yea its on the server name itself, and to view this you have to explicitly navigate to the v2 spec:
![Screenshot 2020-06-09 at 09 31 21](https://user-images.githubusercontent.com/2732945/84125117-ff35ee80-aa33-11ea-8665-aa2239052fa7.png)